app/images/arrow-right-blue.svg

```diff
@@ -0,0 +1,19 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="40" height="40" fill="none">
```

Comment:
I think we could compress this SVG using something like this https://vecta.io/nano> ?

Reply:
Done